addons/web_tour/static/src/scss/tip.scss

```diff
@@ -76,6 +76,32 @@
         &.right, &.left {
             animation-name: move-left-right !important;
         }
+        box-shadow: none;
+        border-color: transparent !important;
+        border-width: $o-tip-size / 2 !important;
+        background-color: transparent !important;
+        background-size: $o-tip-size * 2;
+        background-repeat: no-repeat;
+        background-position-x: 50%;
+        background-position-y: 50%;
+
+        &.bottom {
+            background-image: url('/web_tour/static/src/img/tip_arrow_up.svg');
+        }
+        &.top {
+            background-image: url('/web_tour/static/src/img/tip_arrow_down.svg');
+        }
+        &.right {
+            background-image: url('/web_tour/static/src/img/tip_arrow_left.svg');
+        }
+        &.left {
+            background-image: url('/web_tour/static/src/img/tip_arrow_right.svg');
+        }
```

Comment:
4 svg for 4 directions is just useless, if we make a svg we will just have to rotate it in css for those cases.

Reply:
I did it this way thinking that a designer might deliver distinct images - e.g. for some light effects that would depend on the orientation